packages/client/hmi-client/src/components/sidebar-panel/documents-sidebar-panel.vue

```diff
@@ -67,29 +61,9 @@ onMounted(() => {
 </script>
```

Comment:
I am not sure how to test the functionality of this or if I should.

Reply:
You have to:
* open the _Data Explorer_ (to right green icon).
* go to the _Articles_ page
* select one or more 
* add them to the current project in the right sidebar
* 🎉